src/util/validation/index.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Validates signup fields.
+ *
+ *
+ * @remarks
+ * Could add more validation checks, such as a regex for email.
+ *
+ * @param username - A string for username.
+ * @param email - A string for user's email.
+ * @param password - A string for user's password.
+ * @param confirm - A string to confirm user's password.
+ *
+ * @returns - String (for an error) or null (for no error).
+ *
+ *
+ */
+export const validateSigniupFields = ({
```

Comment:
Great tool, just for the future consider a package for [validation](https://github.com/validatorjs/validator.js). No need to reinvent the wheel😄